.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,28 @@
+### Issue
+
+#### Issue Checklist
+- [ ] Checked [README](README.md) and [CONTRIBUTING](CONTRIBUTING.md)
+- [ ] Checked all existing or past issues
+- [ ] Updated to the latest version
```

Comment:
Not sure "Updated to the latest version" is relevant for this project as I'd expect most people are accessing via [kanban.ist](https://kanban.ist).